cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -0,0 +1,56 @@
+Crontasks settings
+==================
+
+You find here, some good practices about crontasks management in Akeneo PIM Flexibility Edition :
+
+1. CRONs we provide by default uses a custom wrapper script defined by *SHELL* that call the PIM "bin/console" command.
+
+   It avoid us to write on each cron line the full script path to call and the redirection of logs.
+
+   Adding your own cron lines failes because it tries to run your command with our wrapper.
+   
+   To run your own script normally (without our wrapper), you just have to redefine the shell variable like this "SHELL=/bin/bash" before them
```

Comment:
(I don't understand the "before them")

Reply:
To normally run your own..

Reply:
Use ` (backticks) instead of quotes so it gets properly formated ;)

Reply:
`But If you want to run your own script (without the wrapper), you will have to redefine the SHELL variable like this "SHELL=/bin/bash" before using your own custom command`